recipes/pyulog/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pyulog" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "5899de5950ded5890af934c1902f4525a1ec7c48dec4587ebbc345f1aa8db1e6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PX4/pyulog/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
It looks like this package has a number of console scripts:

https://github.com/PX4/pyulog/blob/master/setup.py#L56-L63

Please add them as `entry_points`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

and add the appropriate command line tests:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

Reply:
Hi John, thanks for the review! I added the entry points and the commands.